test/models/adoptions_request_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class AdoptionsRequestTest < ActiveSupport::TestCase
+  test "should not save gem without gems_adoption_id or user_id" do
+    g = AdoptionsRequest.new
+    g.save
```

Comment:
Also I'm not a big fan of single character name variables, do you think you it would be better to use something more descriptive like `gem` instead of `g`?